Packs/TaniumThreatResponse/Playbooks/playbook-Tanium_Threat_ResponseV2_-_Request_File_Download.yml

```diff
@@ -0,0 +1,452 @@
+              simple: "true"
+      label: "yes"
+    id: "10"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#default#':
+      - "8"
+      "yes":
+      - "9"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: If true, the file will be downloaded, otherwise, only a file download
+        object will be created.
+      id: 7b130651-ed76-4bba-88c1-379c85b1b977
+      iscommand: false
+      name: Should the file be downloaded?
+      type: condition
+      version: -1
+    taskid: 7b130651-ed76-4bba-88c1-379c85b1b977
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 895
+        }
+      }
+  "12":
+    id: "12"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "10"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      task_id:
+        simple: ${Tanium.FileDownloadTask.taskId}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Tanium Threat Response v2
+      description: Get task by ID.
+      id: 9a0d1e5e-2211-4417-8a9c-cc8a3987f840
+      iscommand: true
+      name: tanium-tr-get-task-by-id
+      script: Tanium Threat Response v2|||tanium-tr-get-task-by-id
+      type: regular
+      version: -1
+    taskid: 9a0d1e5e-2211-4417-8a9c-cc8a3987f840
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 720
+        }
+      }
+  "14":
+    id: "14"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    loop:
+      exitCondition: ""
+      iscommand: false
+      max: 0
+      wait: 1
+    nexttasks:
+      '#none#':
+      - "1"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      client-id:
+        simple: ${inputs.client-id}
+      hostname:
+        simple: ${inputs.hostname}
+      ip:
+        simple: ${inputs.ip}
+      platform:
+        simple: ${inputs.platform}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: d494ed5d-2afe-44de-83c2-5f94d4234dbf
+      iscommand: false
+      name: 56a4f91c-37f3-48aa-8ac2-e9bbd3c38687
+      playbookId: 56a4f91c-37f3-48aa-8ac2-e9bbd3c38687
+      type: playbook
+      version: -1
+      description: ''
+    taskid: d494ed5d-2afe-44de-83c2-5f94d4234dbf
+    timertriggers: []
+    type: playbook
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 190
+        }
+      }
+version: -1
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1610,
+        "width": 492.5,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+fromversion: 5.0.0
+tests:
+- Tanium Threat Response V2 Test
+- Tanium Threat Response Test
```

Comment:
why do we have both tests here?

Reply:
not sure, changed to the playbook that should test this playbook